### PR TITLE
docs(watermark): fix textAlign defaultValue

### LINKED ISSUE #51209: 文档有误
### Reproduction link

[https://ant.design/components/watermark-cn#font](https://ant.design/components/watermark-cn#font)

### Steps to reproduce

https://ant.design/components/watermark-cn#font

### What is expected?

textAlign 的默认值为 center

### What is actually happening?

textAlign 的默认值为 5.10.0

| Environment | Info |
| --- | --- |
| antd | 5.21.3 |
| React | v18 |
| System | win11 |
| Browser | chrome129 |

image

### PR DESCRIPTION
**Pull request by aojunhao123**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues
fix #51209 

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     fix textAlign defaultValue      |
| 🇨🇳 Chinese |     修正文档中textAlign属性的默认值描述      |

### CODE REVIEW

File: components/watermark/index.zh-CN.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ tag: 5.1.0
 | fontWeight | 字体粗细 | `normal` \| `light` \| `weight` \| number | normal |  |
 | fontFamily | 字体类型 | string | sans-serif |  |
 | fontStyle | 字体样式 | `none` \| `normal` \| `italic` \| `oblique` | normal |  |
-| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | 5.10.0 |
+| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | center |
```
Comment (afc163) [inline components/watermark/index.zh-CN.md:59]:
```suggestion
| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | center | 5.10.0 |
```

Comment (aojunhao123) [reply components/watermark/index.zh-CN.md]: done

**Review by afc163 — APPROVED** [review]

File: components/watermark/index.zh-CN.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ tag: 5.1.0
 | fontWeight | 字体粗细 | `normal` \| `light` \| `weight` \| number | normal |  |
 | fontFamily | 字体类型 | string | sans-serif |  |
 | fontStyle | 字体样式 | `none` \| `normal` \| `italic` \| `oblique` | normal |  |
-| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | 5.10.0 |
+| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | `center` |
```
Comment (afc163) [inline components/watermark/index.zh-CN.md:59]:
```suggestion
| textAlign | 指定文本对齐方向  | [CanvasTextAlign](https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/textAlign) | `center` | 5.10.0 |
```

Comment (aojunhao123) [reply components/watermark/index.zh-CN.md:59]: Thank you for your careful review！done.

**Comment by aojunhao123** [comment]
> > 同一个 issue 被多个人修复了，大家太积极了，本着先来后到原则，当前 PR 应该被采纳。 但是 review 后发现少了一列，再次commit 又要走一遍 CI。
> > 另外一个 PR 没啥问题，并且是 issue 作者本人提交的，所以我采纳了另外一个 PR merge，并将你标记为共同合作者。
> 
> 两位都是 `first-time contributor` 🎉

嗯嗯没事的，以后贡献的机会还有很多😉

**Pull request closed without merging**